src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeMapper.java

```diff
@@ -60,7 +60,7 @@ public static Envelope toDbEnvelope(InputEnvelope envelope, String containerName
         }
     }
 
-    private static ScannableItem toDbScannableItem(InputScannableItem scannableItem) {
+    public static ScannableItem toDbScannableItem(InputScannableItem scannableItem) {
```

Comment:
Why this?

Reply:
to be able to test just this mapping in separation

Reply:
Then maybe we should limit it to default accessor, as it's not really used by anything other than this class?